pkg/federation/sdlmerge/remove_duplicate_fielded_shared_types.go

```diff
@@ -109,15 +109,15 @@ func (r *removeDuplicateFieldedSharedTypesVisitor) LeaveDocument(_, _ *ast.Docum
 
 func (r *removeDuplicateFieldedSharedTypesVisitor) checkForDuplicateEntity(hasDirectives bool, directiveRefs []int, name string) {
 	if r.entitySet[name] {
-		r.StopWithExternalErr(operationreport.ErrEntitiesMustNotBeSharedTypes(name))
+		r.StopWithExternalErr(operationreport.ErrEntitiesMustNotBeDuplicated(name))
```

Comment:
Missing `return` or `break` statement?

Reply:
I am not sure what you mean for this one.